functions.php

```diff
@@ -501,3 +501,15 @@ function ($canonical_url, $post) {
     10,
     2
 );
+
+// This filter overrides the default revisions value which by default it is set to 1.
+// That means that all revisions are stored and the db storage it's growing unnecessarily.
+add_filter(
+    'wp_revisions_to_keep',
+    function () {
+        $revisions_to_keep = planet4_get_option('revisions_to_keep', 5);
```

Comment:
We didn't define specific requirements in the ticket but maybe `5` is too low. Maybe `20` would be a more safe default options for editors?